exp/api/v1beta2/awsmachinepool_webhook_test.go

```diff
@@ -153,6 +153,27 @@ func TestAWSMachinePoolValidateCreate(t *testing.T) {
 			},
 			wantErr: true,
 		},
+		{
```

Comment:
Maybe a test for when the value is above 200?

Reply:
and below 100

Reply:
This is just cherry picking from upstream, so I'd prefer not to change the tests just in the fork.